dockerfiles/base/Dockerfile

```
@@ -8,7 +8,7 @@ ARG NODE_VERSION
 ENV DEBIAN_FRONTEND=noninteractive
 ENV NODE_VERSION=${NODE_VERSION}
 ENV PATH=$PATH:/home/node/.npm-global/bin
-ARG USER_UID=1000
+ARG USER_UID=2000
```

The reference why we have to bump user_id: https://bugs.launchpad.net/cloud-images/+bug/2005129

---

ugh... that's not that great...

---

Adds more stuff here, but:

```suggestion
ARG USER_UID=1000

RUN userdel ubuntu && groupadd -g $USER_UID snyk && useradd -u $USER_UID -g $USER_UID --no-create-home
```

---

@soniqua, just fy: `ubuntu` user has id 1000 and this user, so creating node user with id 1000 will give an ownership to these folder.

We checked other official Docker images based on `ubuntu:24.04`, so we see ppl create a new user (instead of deleting an ubuntu one).